src/coreclr/src/System.Private.CoreLib/src/System/Array.CoreCLR.cs

```diff
@@ -483,6 +483,7 @@ private unsafe bool IsValueOfElementType(object value)
         public extern void Initialize();
     }
 
+#pragma warning disable CA1822
```

Comment:
```suggestion
#pragma warning disable CA1822 // Mark members as static
```
Same in other places.